src/main/java/org/beanpod/switchboard/config/WebConfig.java

```diff
@@ -0,0 +1,15 @@
+package org.beanpod.switchboard.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Configuration
+public class WebConfig {
+
+  @Bean
+  public BCryptPasswordEncoder bCryptPasswordEncoder() {
```

Comment:
To reduce the code smell related to `Method name 'bCryptPasswordEncoder' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'`. I'd suggest refactoring `bCryptPasswordEncoder` --> `bcryptPasswordEncoder` or `passwordEncoder`. Any name such that the second character is a lowercase letter, or number would be compliant.

Reply:
fixed